.github/workflows/review.yaml

```
@@ -0,0 +1,87 @@
+        echo export LD_LIBRARY_PATH="$BASEDIR/lib" >> src/build/activate_env.sh
+
+    # Step to package the build directory
+    - name: Create tar.gz package
+      run: |
+        BUILD_DIR="src/build"
+        PACKAGE_NAME="OpenConverter_Linux_x86.tar.gz"
+        tar -czvf $PACKAGE_NAME -C $BUILD_DIR .
+
+    # Step to upload the tar.gz package as an artifact
+    - name: Upload build artifact
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)